app/presenters/ht_registration_presenter.rb

```diff
@@ -4,7 +4,12 @@ class HTRegistrationPresenter < ApplicationPresenter
   ALL_FIELDS = %i[
     dsp_name dsp_email dsp_date inst_id jira_ticket
     auth_rep_name auth_rep_email auth_rep_date
-    contact_info mfa_addendum sent received finished ip_address env
+    contact_info mfa_addendum sent received finished ip_address
+  ].freeze
+
+  DETAIL_FIELDS = %i[
+    detail_edu_person_principal_name detail_display_name detail_geoip
```

Comment:
I would like to also see the mail and eduPersonScopedAffiliation attributes mapped here.

Reply:
Addressed in next commit